src/core/crypto.c

```diff
@@ -1719,8 +1724,42 @@ QuicCryptoCustomCertValidationComplete(
             "Custom cert validation failed.");
         QuicConnTransportError(
             QuicCryptoGetConnection(Crypto),
-            QUIC_ERROR_CRYPTO_ERROR(0xFF & CXPLAT_TLS_ALERT_CODE_BAD_CERTIFICATE));
+            QUIC_ERROR_CRYPTO_ERROR(0xFF & CXPLAT_TLS_ALERT_CODE_BAD_CERTIFICATE)); //
+    }
+}
+
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QuicCryptoCustomTicketValidationComplete(
+    _In_ QUIC_CRYPTO* Crypto,
+    _In_ BOOLEAN Result
+    )
+{
+    if (Crypto->TicketValidationStatus != QUIC_SERVER_VALIDATION_PENDING) {
+        return;
+    }
+
+    if (Result) {
+        Crypto->TicketValidationStatus = QUIC_SERVER_VALIDATION_DONE;
+        QuicCryptoProcessDataComplete(Crypto, Crypto->PendingValidationBufferLength);
```

Comment:
`QuicCryptoCustomCertValidationComplete` should also be updated to call this now.

Reply:
wow, kind of bug😮